### PR TITLE
Prepare stewardci-core for improved release process

### PR DESCRIPTION
**Pull request by alxsap**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<!-- add links to release notes if important dependencies changed -->
+N/A
+
+### Submitter checklist
+
+- [ ] Change has been tested (on a back-end cluster)
+- [ ] [changelog.yaml] with upgrade notes are prepared and appropriate for the audience affected by the change (users or developer, depending on the change).
+- [ ] Semantic version diffed against [last release][releases] and updated accordingly. In this project the version has to be maintained here:
+  - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml)
```
Comment (rinckm) [inline .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
  - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml)
      - version: 
      - appVersion: 
```

Comment (rinckm) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: Both?

Comment (alxsap) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: We always release the steward controller images and Helm chart together, right? So yes, both. I'll add it.

Comment (alxsap) [reply .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
  - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) (`version` and `appVersion`)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+### Dependency release notes
+
+<!-- add links to release notes if important dependencies changed -->
+N/A
+
+### Submitter checklist
+
+- [ ] Change has been tested (on a back-end cluster)
```
Comment (rinckm) [inline .github/PULL_REQUEST_TEMPLATE.md:15]:
```suggestion
- [ ] Change has been tested (on a back-end cluster)
  Get system information with following commands:
  - Controllers: 'kubectl -n steward-system get deployments.apps -o yaml | grep "image:"'
    JFR: 'kubectl get clustertasks.tekton.dev steward-jenkinsfile-runner -oyaml | grep "image:"'
    HELM: 'helm -n steward-system list'
    
    
```

Comment (alxsap) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: This should be documented in a README and we would simply link to this. We should not describe it in the PR template.

**Review by rinckm — APPROVED** [review]

**Pull request merged**